### ISSUE TITLE
Calcium binding to lipid bilayers with PG lipids

### ISSUE THREAD

**Issue by ohsOllila** [NMRLipids/NMRlipidsIVPEandPG#12]
Conclusions on ion binding to PG containing bilayers are still unclear.

Currently, we have results from POPC:POPG (1:1) and POPC:POPG (4:1) mixtures simulated with CHARMM36 and Lipid17 force fields with different CaCl2 concentrations. 

**CHARMM36 simulations** were generated with CHARMM-GUI [before the NBfix was incorporated for calcium](http://nmrlipids.blogspot.com/2017/12/nmrlipids-iv-current-status-and.html?showComment=1519847165408#c2532497549973755104). In [the current figure](https://github.com/NMRLipids/NMRlipidsIVPEandPG/blob/master/Figs/CHANGESwithCaClPG-eps-converted-to.pdf), the response of order parameters to increasing calcium concentration seems to be actually quite close to the experimental data in these simulations. However, there are two issues here:

1) The simulations are only 200ns long. In supplementary information of NMRlipids II publication,
the amount of bound calcium was still chancing after one microsecond in CHARMM36 simulations. Also here, the numbers of contacts between lipids and calcium are still increasing after the 200 ns simulation in both [POPC:POPG (1:1)](https://github.com/NMRLipids/MATCH/blob/master/Data/Lipid_Bilayers/POPG%2B50%25POPC%2B1000mMCaCl/T298K/MODEL_CHARMM36/numcont.jpg) and [POPC:POPG (4:1)](https://github.com/NMRLipids/MATCH/blob/master/Data/Lipid_Bilayers/POPG%2B80%25POPC%2B1000mMCaCl/T298K/MODEL_CHARMM36/numcont.jpg) mixtures simulated with CHARMM36 and 1 M of CaCl.

2) The system without calcium was ran using potassium counterions. However, the potassium counterions were not present in simulations with CaCl2 where PG charge was balanced with calcium. Originally the simulations without monovalent counterions were made to focus on calcium-binding without disruptions from incorrect counter-ion binding. However, I have now realized that there are quite a lot of counterions in the systems with negatively charged lipids, especially in 1:1 POPC:POPG mixtures. Therefore, most calcium in simulations with approximately 100mM CaCl concentrations act as "counter-ions" and it is difficult to decide what would be the calcium concentration corresponding to experiments in these systems.

**Lipid17 simulations** of the same mixtures are also contributed with the exception that sodium counterions are present also in systems with added calcium:
https://zenodo.org/record/3520479#.XbmWti17FBw
https://zenodo.org/record/3520499#.XbmWzC17FBw
https://zenodo.org/record/3520503#.XbmW9S17FBw

https://zenodo.org/record/3516644#.XbmXDS17FBw
https://zenodo.org/record/3516913#.XbmXRy17FBw
https://zenodo.org/record/3516710#.XbmXLC17FBw

However, POPC:POPG (1:1) mixtures in these simulations do not seem to behave very well: Without CaCl2 one POPG departs from bilayer to the water phase, water pore containing also calcium ions occurs [in simulation with 100mM CaCl2](https://github.com/NMRLipids/MATCH/blob/master/Data/Lipid_Bilayers/POPG%2B50%25POPC%2B100mMCaCl/T298K/MODEL_LIPID17/finalSTRUCT.tga), and bilayer with 1 M CaCl [is not stable](https://github.com/NMRLipids/MATCH/blob/master/Data/Lipid_Bilayers/POPG%2B50%25POPC%2B1000mMCaCl/T298K/MODEL_LIPID17/finalSTRUCT.tga). Such issues are not seen in POPC:POPG (4:1) mixtures, but the amount of bound calcium ions is [not equilibrated also in these systems](https://github.com/NMRLipids/MATCH/blob/master/Data/Lipid_Bilayers/POPG%2B80%25POPC%2B1000mMCaCl/T298K/MODEL_LIPID17/numcont.jpg). However, the [order parameter changes](https://github.com/NMRLipids/NMRlipidsIVPEandPG/blob/master/Figs/CHANGESwithCaClPG-eps-converted-to.pdf) and [density profiles](https://github.com/NMRLipids/NMRlipidsIVPEandPG/blob/master/Figs/CAdensPG-eps-converted-to.pdf) suggest that there are more calcium bound than in CHARMM36 simulations and that the order parameter changes are larger than in experiments. It is not clear if the above mentioned bad behavior of POPC:POPG (1:1) mixtures in Lipid17 simulations is because of the too strong ion binding or due to some other properties of this force field.

In conclusion, I think that we should at least **run longer simulations with CHARMM using the current NBfix from CHARMM-GUI and sodium counterions also when CaCl is added**. Also **results from other force fields would be highly useful**. If someone is willing to run the new CHARMM36 simulations or deliver data from other force fields, let me know.

**Comment by jmelcr** [comment]
I have taken a closer look at the simulations with Lipid17 POPG and compared the used topologies with those used previously. 
For instance, the [POPC topology](https://zenodo.org/record/2585523/files/gromacs.top?download=1) generated with the mentioned `ambertogromacs.py` script in [this Zenodo repository](https://zenodo.org/record/2585523/) (used in the above mentioned simulations) does not agree with the previously used [POPC topology](https://github.com/jmelcr/ecc_lipids/blob/master/topologies/ecc-lipids/ecc-popc/ECC-POPC_amber-lipid14-atomic-names.itp) we created for [PC-Ca2+ paper](https://github.com/ohsOllila/NMRlipids_VI-NewIonModel/blob/master/Manuscript/manuscript.pdf). 

Apart from a few angles being slightly different and partial charges slightly updated in head groups and connections to them (which may reflect the change from Lipid14->17) there is one clear difference in the topology: the dihedrals are chosen as **type 1**, while the correct type shall be **type 9**. 
Both dihedral types 1 and 9 use the same interaction form, but the major difference between them is that when there are several lines for one dihedral (which is the case for Amber potentials), type 1 “rewrites the potential to the last appearing record in the topology", while type 9 "sums the lines together” considering them as individual contributions, as is described in [Gromacs manual, page 81, bottom](http://manual.gromacs.org/documentation/2018-current/manual-2018.8.pdf).  

This mis-assignment of dihedral types is a mistake made by `ParmEd` (used here, but can be easily corrected if known), but not made by `acpype`. From my past tests with phospholipids, other parameters were transformed the same by both scripts.

I have corrected the [POPG topology to use dihedral type 9, here](https://github.com/jmelcr/ecc_lipids/blob/master/topologies/lipid17/lipid17-POPG_amber-atomic-names_and_order.itp). Topologies for [POPC](https://github.com/jmelcr/ecc_lipids/blob/master/topologies/ecc-lipids/ecc-popc/ECC-POPC_amber-lipid14-atomic-names.itp) (lipid14) and [CHOL](https://github.com/jmelcr/ecc_lipids/blob/master/topologies/lipid17/lipid14-CHOL_amber-order.top) (discussed in [issue #7 in NMRlipids proj. III](https://github.com/NMRLipids/NmrLipidsCholXray/issues/7)) are also there in the same [respository](https://github.com/jmelcr/ecc_lipids/tree/master/topologies).

I hope this helps solving the troubles with POPG and Ca2+!

**Comment by ohsOllila** [comment]
Thanks for pointing this out. I believe that using type 1 instead of type 9 in dihedral might lead to significantly different results. I will rerun the above listed simulations with calcium using the correct dihedral type. In addition, it might be a good idea to rerun [the pure POPG simulation](https://doi.org/10.5281/zenodo.3247659) by @peonqan with the type 9 dihedral to check if the underestimated area per molecule could be explained by this.

**Comment by peonqan** [comment]
Hello, what dihedral angles should be changed?
Antonio Peón

**Comment by ohsOllila** [comment]
In my understanding, all dihedrals with type 1 should be changed to type 9.

I have made this for the POPC:POPG (1:1) mixtures with different calcium concentrations above and the simulations are now in a queue in the cluster. The modified *top file is here:
https://www.dropbox.com/s/6e9jlz50fty0kky/gromacsFIXED.top?dl=0
 for the system without additional CaCl2 (original data in here: https://doi.org/10.5281/zenodo.3520478).

**Comment by peonqan** [comment]
Thanks, I relaunched the POPG without salt.  I modified the dihedral
function type to 9 and the sodium parameters as in your topology file.
Na_d         11 22.98977    1.000      A     0.235000        0.54390000 ;
DOI: 10.1063/1.466363  ; Dang1994
Antonio Peón

**Comment by ohsOllila** [comment]
I have now run 720ns simulations of POPC:POPG (1:1) mixture with LIPID17 with corrected dihedral parameters, and with 100mM and 1000mM CaCl2. The obvious artifacts mentioned above are not present anymore and the amount of calcium seemed to be equilibrated during the last 200ns of the trajectory. I have now added the results analyzed from the last 200ns to the plots in the manuscript. The simulations of POPC:POPG (4:1) mixtures with corrected dihedral, and with different calcium concentrations are still running.

**Comment by ohsOllila** [comment]
I have now also updated the results from  POPC:POPG (4:1) mixtures with the corrected dihedral. The data in Zenodo repositories need to be still updated. 

I am now in process of adding also the data from Slipids force field provided by Matti Javanainen: [http://doi.org/10.5281/zenodo.3613573](http://doi.org/10.5281/zenodo.3613573) to the manuscript.

**Comment by peonqan** [comment]
Hello,

I  have run the POPG (LIPID17 FF) simulation with the corrected dihedral a
and these are the results. There's not much variation from the previous
one.

# OP_name    resname    atom1    atom2    OP_mean   OP_stddev  OP_stem
#--------------------------------------------------------------------
gamma1               PGR     C33   H3A    0.00993  0.07997  0.00358
gamma2               PGR     C33   H3B    0.01309  0.07960  0.00356
beta1                PGR     C32   H2A    0.00001  0.10438  0.00467
alpha1               PGR     C31   H1A    0.10765  0.12098  0.00541
alpha2               PGR     C31   H1B    0.09967  0.12529  0.00560
g3_1                 PGR     C3    HA    -0.11162  0.16818  0.00752
g3_2                 PGR     C3    HB    -0.18401  0.14035  0.00628
g2_1                 PGR     C2    HX    -0.22936  0.13954  0.00624
g1_1                 PGR     C1    HR    -0.22819  0.15041  0.00673
g1_2                 PGR     C1    HS     0.05972  0.15811  0.00707

The difference is:

gamma1 C33 -0,003
gamma2 C33 -0,005
beta1 C32 -0,009
alpha1 C31 -0,007
alpha2 C31 0,003
g3_1 C3 -0,004
g3_2 C3 0,004
g2_1 C2 -0,014
g1_1 C1 -0,003
g1_2 C1 0,012


Antonio Peón

**Comment by ohsOllila** [comment]
Hello @peonqan,

could you update the POPG simulation data with the correct dihedral into the Zenodo repo: https://doi.org/10.5281/zenodo.3247658. 

I addition, we need also Lipid17 simulations of POPG with additional salt and POPE simulations (with and without additional salt) with the correct dihedrals. These simulations are currently in these repositories:
https://doi.org/10.5281/zenodo.2573904
https://doi.org/10.5281/zenodo.3237700
https://doi.org/10.5281/zenodo.2577304

**Comment by peonqan** [comment]
Hello,

I updated the link with the dihedral type 1:
https://doi.org/10.5281/zenodo.3247658

Then, I'll run the others simulations.



On Fri, 15 May 2020 at 15:36, ohsOllila <notifications@github.com> wrote:

> Hello @peonqan <https://github.com/peonqan>,
>
> could you update the POPG simulation data with the correct dihedral into
> the Zenodo repo: https://doi.org/10.5281/zenodo.3247658.
>
This is the new one for LIPID17 POPG using dihedral type 1 instead of 9:
https://doi.org/10.5281/zenodo.3832219

I addition, we need also Lipid17 simulations of POPG with additional salt

**Comment by ohsOllila** [comment]
Thanks @peonqan. 

I have now updated the results from corrected data into the manuscript. However, I am still little bit confused with the Zenodo entries. I guess that here https://doi.org/10.5281/zenodo.3832219 the title should be "type 9 instead of type 1". Another thing is that also the original https://doi.org/10.5281/zenodo.3247658 has actually the popg_fix.top with the dihedral type 9. Did you update this file also in there?

**Comment by ohsOllila** [comment]
Lipid17 results with correct dihedral types and Slipids results are now included into the manusript [(Figs. 9](https://github.com/NMRLipids/NMRlipidsIVPEandPG/blob/master/Figs/CHANGESwithCaClPG1PC1-eps-converted-to.pdf) and [S6](https://github.com/NMRLipids/NMRlipidsIVPEandPG/blob/master/Figs/CHANGESwithCaClPG1PC4-eps-converted-to.pdf). 

I also applied the electronic continuum correction (see http://dx.doi.org/10.1021/acs.jctc.9b00824 and http://dx.doi.org/10.1021/acs.jpcb.7b12510) to Lipid17 POPG parameters and ran the simulations with these parameters (in the same figures as above). The binding affinity based on PC headgroup order parameter changes seems more reasonable in the ECC-lipid model (as observed previously), but the response of PG beta carbon order parameter to calcium did not improve (actually got worse). 

We still need longer CHARMM36 simulations with Na+ counterions and ECC-lipid simulations from POPC (4:1) mixtures. These simulations are in progress in my group.

**Comment by ohsOllila** [comment]
> Hello @peonqan,
> 
> could you update the POPG simulation data with the correct dihedral into the Zenodo repo: https://doi.org/10.5281/zenodo.3247658.
> 
> I addition, we need also Lipid17 simulations of POPG with additional salt and POPE simulations (with and without additional salt) with the correct dihedrals. These simulations are currently in these repositories:
> https://doi.org/10.5281/zenodo.2573904
> https://doi.org/10.5281/zenodo.3237700
> https://doi.org/10.5281/zenodo.2577304

In addition to the systems mentioned here, we need also this dataset simulated with the correct dihedral potential: https://doi.org/10.5281/zenodo.3241242

And also probably this one: https://doi.org/10.5281/zenodo.3237656, which seems to contain an inconsistent top file.

**Comment by peonqan** [comment]
Hello,
The two links are the same.
Antonio Peón

**Comment by peonqan** [comment]
Hello,
The POPG simulation with salt is updated and the others simulations are
running.
Antonio Peón

**Comment by ohsOllila** [comment]
Hello @peonqan,

the focus of the manuscript is slightly updated (see http://nmrlipids.blogspot.com/2020/12/nmrlipids-ivb-toward-submission-of.html) and these simulations are now in the supplementary information. It is therefore not absolutely necessary to include these, but if you already have the above mentioned simulations ran with correct dihedrals, it would make sense to put these into the Zenodo and the results in the supplementary information.

**Comment by peonqan** [comment]
Hi,
Here are the new updates uploaded.

POPE+ 150 mM NaCl
https://zenodo.org/record/4424934#.X_gfh9Z7nLA
POPE
https://zenodo.org/record/4424292#.X_gfFtZ7nLA
POPG + 150 mM NaCl
https://zenodo.org/record/4386514#.X_gfF9Z7nLA

Best regards,

Antonio

**Comment by ohsOllila** [comment]
I have now updated all simulations with correct dihedrals into the manuscript and I will close this issue.
